oauthproxy.go

```diff
@@ -1248,3 +1255,27 @@ func (p *OAuthProxy) errorJSON(rw http.ResponseWriter, code int) {
 	// application/json
 	rw.Write([]byte("{}"))
 }
+
+// LoggingCSRFCookiesInOAuthCallback Log all CSRF cookies found in HTTP request OAuth callback,
+// which were successfully parsed
+func LoggingCSRFCookiesInOAuthCallback(req *http.Request, cookieName string) {
```

Comment:
Does this really need to be exported?

Reply:
I think I've done this due to LINT. Methods should not have more than a couple of lines.
Any suggestions?

Reply:
Definitely should not be exported, please make it a private method and if the linter is complaining, let me know and I can help you work out how to resolve it